apps/src/music/views/ChordPanel.tsx

```diff
@@ -116,6 +116,13 @@ const ChordPanel: React.FunctionComponent<ChordPanelProps> = ({
     setIsPlayingPreview(false);
   }, [setIsPlayingPreview]);
 
+  useEffect(() => {
+    // On unmount.
+    return () => {
+      stopPreview();
+    };
+  }, [stopPreview]);
```

Comment:
Optional: might be able to inline this to `useEffect(() => stopPreview, [stopPreview]);`